schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -286,6 +289,21 @@
                 "ui"
             ],
             "title": "ServiceInfo"
+        },
+        "Control": {
+            "type": "object",
+            "description": "Device settings configuration",
+            "properties": {
+                "alertsEn": {
```

Comment:
Why do we need `alertsEn` when we have `logLvl` `0`?

Reply:
Those are separate mechanisms.  AlertsEn is for the alerts system in the device, which is a new creation -- it is not related to the existing Zephyr logging system.

LogLvl is for controlling the logging system in the device, and it is there that log level 0 means off.

We want independent control over alerts vs. logs.

Reply:
alerts and logs are different